playbooks/adhoc/create_pv/create_pv.yaml

```diff
@@ -0,0 +1,102 @@
+---
+# example run: 
+# ansible-playbook -e "volume_size=1" \
+#                  -e "device_name=/dev/xvdf" \
+#                  -e "oo_group=tag_env-host-type_ops-openshift-master" \
+#                  create_pv.yaml
+#
+- name: "populate oo_master"
+  hosts: localhost
+  gather_facts: no
+  vars:
+    volume_size: "{{ volume_size }}"
+    device_name: "{{ device_name }}"
+    iops: "{{ volumes_iops }}"
+    oo_group: "{{ groups[oo_group] }}"
+  tasks:
```

Comment:
Need fail module calls for required vars:

https://github.com/twiest/openshift-ansible/blob/master/docs/best_practices_guide.adoc#defensive-programming